### PR TITLE
refactor: resolve a few TODOs

### PR DESCRIPTION
**Pull request by sillyElvy**
This PR resolves a few TODOs

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [ ] I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

File: interactions/api/error.py [hunk 1]
```diff
@@ -261,11 +261,112 @@ def lookup() -> dict:
...
             20022: "This message cannot be edited due to announcement rate limits.",
             20028: "The channel you are writing has hit the write rate limit",
-            20031: "Your Stage topic, server name, server description, or channel names contain words that are not allowed",
-            20035: "Guild premium subscription level too low"
-            #  TODO: post-v4: finish dict.
+            20031: "Your Stage topic, server name, server description, "
+            "or channel names contain words that are not allowed",
+            20035: "Guild premium subscription level too low",
+            30001: "Maximum number of guilds reached (100)",
+            30002: "Maximum number of friends reached(1000)",
```
Comment (AstreaTSS) [inline interactions/api/error.py]: You forgot a space here for max friends, haha.

Comment (sillyElvy) [reply interactions/api/error.py]: Actually that was pre-commit

File: interactions/api/models/misc.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # TODO: This is post-v4.
-# TODO: Reorganise these models based on which big obj uses little obj
-# TODO: Potentially rename some model references to enums, if applicable
-# TODO: Reorganise mixins to its own thing, currently placed here because circular import sucks.
+#   Reorganise these models based on which big obj uses little obj
+#   Potentially rename some model references to enums, if applicable
```
Comment (i0bs) [inline interactions/api/models/misc.py:3]: Have these TODOs actually been completed? Removing the convention doesn't suddenly make it no longer a task.

File: interactions/api/error.py (hunk 1 above)
Comment (sillyElvy) [reply interactions/api/error.py]: oh there
I see now

Comment (Astrea49) [reply interactions/api/error.py:272]: ??? `pre-commit` touches many things but not strings, at least not to the point where it would remove a space.

Comment (sillyElvy) [reply interactions/api/error.py]: I thought of something different, nvm

File: interactions/api/models/misc.py (hunk 2 above)
Comment (sillyElvy) [reply interactions/api/models/misc.py:3]: No, but you only need one TODO-statement. If you indent the other lines, they will be counted as a whole TODO-statement

Comment (sillyElvy) [reply interactions/api/models/misc.py:3]: grafik

File: interactions/api/error.py [hunk 3]
```diff
@@ -261,11 +261,112 @@ def lookup() -> dict:
...
+            30030: "Maximum number of server categories has been reached",
+            30031: "Guild already has a template",
+            30033: "Max number of thread participants has been reached (1000)",
+            30035: "Maximum number of bans for non-guild members have been exceeded",
+            30037: "Maximum number of bans fetches has been reached",
+            30038: "Maximum number of uncompleted guild scheduled events reached (100)",
+            30039: "Maximum number of stickers reached",
+            30040: "Maximum number of prune requests has been reached. Try again later",
+            30042: "Maximum number of guild widget settings updates has been reached. Try again later",
+            30046: "Maximum number of edits to messages older than 1 hour reached. Try again later",
```
Comment (Toricane) [inline interactions/api/error.py:293]: Why do some of these (the ones starting with "Maximum") have a number specified and some don't?

File: interactions/api/error.py [hunk 4]
```diff
@@ -261,11 +261,112 @@ def lookup() -> dict:
...
+            60003: "Two factor is required for this operation",
+            80004: "No users with DiscordTag exist",
+            90001: "Reaction was blocked",
+            130000: "API resource is currently overloaded. Try again a little later",
+            150006: "The Stage is already open",
+            160002: "Cannot reply without permission to read message history",
+            160004: "A thread has already been created for this message",
+            160005: "Thread is locked",
+            160006: "Maximum number of active threads reached",
+            160007: "Maximum number of active announcement threads reached",
```
Comment (Toricane) [inline interactions/api/error.py:362]: Same with these

**Review by Toricane — CHANGES_REQUESTED** [review]
Why do some of these (the ones starting with "Maximum") have a number specified and some don't?

File: interactions/api/error.py (hunk 3 above)
Comment (sillyElvy) [reply interactions/api/error.py:293]: Ddocs, I just copied their info

Comment (EdVraz) [reply interactions/api/error.py:293]: Ddocs, I just copied their info

File: interactions/api/error.py (hunk 4 above)
Comment (sillyElvy) [reply interactions/api/error.py:362]: See above

File: interactions/api/error.py (hunk 3 above)
Comment (Toricane) [reply interactions/api/error.py:293]: Ok then

File: interactions/api/error.py (hunk 4 above)
Comment (Toricane) [reply interactions/api/error.py:362]: Ok

**Review by Toricane — APPROVED** [review]
LGTM!

**Pull request merged**